docs/docs/migration-guide.mdx

```diff
@@ -37,6 +37,7 @@ to the new `YAML` format using following commands:
 
 ```bash
 rasa data convert nlu -f yaml --data={SOURCE_DIR} --out={TARGET_DIR}
+rasa data convert nlg -f yaml --data={SOURCE_DIR} --out={TARGET_DIR}
```

Comment:
i guess if we're adding this, then above you should also say that you can convert responses.md with that command?